components/Footer/Footer.js

```diff
@@ -0,0 +1,99 @@
+        <div className="col-span-1 lg:col-span-2">
+          <Link href="/">
+            <a className="flex flex-initial items-center font-bold md:mr-24">
+              <span className="rounded-full border border-gray-700 mr-2">
+                <Logo />
+              </span>
+              <span>ACME</span>
+            </a>
+          </Link>
+        </div>
+        <div className="col-span-1 lg:col-span-2">
+          <ul className="flex flex-initial flex-col md:flex-1">
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Home
+                </a>
+              </Link>
+            </li>
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Careers
+                </a>
+              </Link>
+            </li>
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Blog
+                </a>
+              </Link>
+            </li>
+          </ul>
+        </div>
+        <div className="col-span-1 lg:col-span-2">
+          <ul className="flex flex-initial flex-col md:flex-1">
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Terms of Use
+                </a>
+              </Link>
+            </li>
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Shipping Returns
+                </a>
+              </Link>
+            </li>
+            <li className="py-3 md:py-0 md:pb-4">
+              <Link href="/">
+                <a className="text-primary hover:text-accents-6 transition ease-in-out duration-150">
+                  Privacy Policy
+                </a>
+              </Link>
+            </li>
+          </ul>
+        </div>
+        <div className="col-span-1 lg:col-span-6 flex items-start lg:justify-end text-primary">
+          <div className="flex space-x-6 items-center h-10">
+            <a
+              aria-label="Github Repository"
+              href="https://github.com/vercel/nextjs-subscription-payments"
+              className={s.link}
+            >
+              <GitHub />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div className="py-12 flex flex-col md:flex-row justify-between items-center space-y-4">
+        <div>
+          <span>&copy; 2020 ACME, Inc. All rights reserved.</span>
```

Comment:
QQ: Is ACME, Inc affiliated with Vercel at all, or is it a well known mock-caompany?